action.yml

```diff
@@ -56,10 +56,12 @@ runs:
       working-directory: ${{ inputs.directory }}
       run: opensafely run run_all --continue-on-error --timestamps --format-output-for-github
     - name: Parse stats logs
+      if: ${{ env.HONEYCOMB_API_KEY }}
```

Comment:
I can't quite work out from github's docs what this does if the env var is defined but empty (as I think it will end up being if someone clones the research-template outside of the opensafely org). It may well treat empty string as false, just wanted to check.

I guess you could set it empty in the default env here and see what happens.

Reply:
I did add a test for running the action without the env variable set.  I add one for when it's but without a value too

Reply:
Yup, looks like a defined but empty var is treated as false too

Reply:
Great, sorry for being anal!